ffmpeg/lpms_ffmpeg.c

```diff
@@ -834,7 +835,10 @@ static int open_input(input_params *params, struct input_ctx *ctx)
   ret = avio_open(&ic->pb, inp, AVIO_FLAG_READ);
   if (ret < 0) dd_err("demuxer: Unable to open file\n");
   ret = avformat_open_input(&ic, NULL, NULL, NULL);
-  if (ret < 0) dd_err("demuxer: Unable to open input\n");
+  if (ret < 0) {
+    avio_closep(&ic->pb); // need to close manually
```

Comment:
This will crash because `ic` is deallocated if `avformat_open_input` fails. In fact, CI is complaining about it.

Also, any error cleanup should go within `open_input_err`

Reply:
Fixed